index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="flex-container header-items">
+    <a href="#" id="header-welcome-link">Welcome</a>
+    <img src="images/union.png" alt="nav button">
+  </header>
+  <section class="flex-container bg-img">
+    <div>
+      <p class="headline-paragraphs" id="p1">Hey there. I’m Mateo<br><span id="p3">I'm a software developer</span></p> 
+      <p class="headline-paragraphs" id="p2">I can help you build a product, feature or website Look through 
+        some of my work and experience! If you like what you see and have 
+        a project you need coded, don’t hestiate to contact me. </p> 
+    </div>
+    <nav class="flex-container nav">
+      <ul>
+        <li>
+          <a href="#">
+            <img src="images/socialLinks/twitter.svg" alt="Twitter Link">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="images/socialLinks/linkedin.svg" alt="Linked Link">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="images/socialLinks/medium.svg" alt="Medium Link">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="images/socialLinks/github.svg" alt="Github Link">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="images/socialLinks/angel.svg" alt="Angel Link">
+          </a>
+        </li>
+      </ul>
+    </nav>
+    <a href="#" class="bttn"><img src="img_girl.jpg" id="bttn-img" alt="Go to Projects" width="10" height="10"></a>
```

Comment:
The arrow is not showing 
![wrong2](https://user-images.githubusercontent.com/58376020/124959101-2ab25080-e01b-11eb-8f86-7874bef67ce0.png)

Reply:
Yeah sorry for that silly mistake :confused: I'ts already fixed :nerd_face: 
Thanks for your help @Shaher-11!!